### DISCUSSION TITLE
Is there a way to add custom dimensions and values to the AWS CloudWatch Metrics sink?

### DISCUSSION THREAD

**Discussion by mkwan-amzn** [vectordotdev/vector#21476] [Q&A] [answered]
Hi folks, 

I'm trying to understand if there's any out-of-the-box way to provide custom dimension keys and values to the Vector metrics that I am sending over. It doesn't seem like the CW Metrics sink itself supports this, nor does the internal metrics source I'm using to get the metrics.

One possibility could be a remap transform step that can somehow add this in but I will need to POC this. I'm wondering if anyone in the community here has experience with this.

**Comment by jszwedko** [comment, marked as answer]
Using `remap` would be the right way. Any tags you add to the metric will be mapped to dimensions in the CloudWatch sink. E.g.

```
.tags.foo = "bar`
```
